src/main/java/io/buildpacks/example/sample/CalculatorService.java

```diff
@@ -23,4 +23,35 @@ public int divide(int a, int b) throws IllegalArgumentException {
         }
         return a / b;
```

Comment:
2. Error 2: Unused private method:
   - The `unusedPrivateMethod` is declared as a private method but is never called within the class. It should be removed if not needed.